pkg/didcomm/protocol/exchange/exchange.go

```diff
@@ -86,29 +91,39 @@ func GenerateInviteWithKeyAndEndpoint(invite *Invitation) (string, error) {
 	return encodedExchangeInvitation(invite)
 }
 
+// Client for exchange protocol
+type Client struct {
```

Comment:
I think recycling this pattern from the Fabric SDK is good (where each subfolder of Client contained a Client). 

I'm wondering if Client is the right word in this case (vs say Protocol or ...)? Thoughts?

Reply:
Makes sense to use Protocol. [Updated](https://github.com/hyperledger/aries-framework-go/compare/f4a9ffc938e46fb640514954eef882c1518e1871..463122c87b11e27a303f59b42906628162e71b73#diff-4d3ca323f9b237cfa9940c12efa9b615R95) the code.